CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
 # Changelog
 
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
+
+## [0.0.1] - 2022-03-29
```

Comment:
@kabilar Should our semantic versioning (while we still use it) reflect our move away from a beta-release cycle? If yes, ignore the suggestion below
```suggestion
## 0.0.0b1 - 2022-03-29
```